docs/install/Knative-custom-install.md

```diff
@@ -391,6 +395,8 @@ commands below.
        --filename https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml \
        --filename https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/config/build/clusterrole.yaml
      ```
+     > **Note**: You can add `-l networking.knative.dev/certificate-provider!=cert-manager` to above `kubctl apply` command to 
```

Comment:
Please revise (globally) to:

> **Optional**: If you do not intend on enabling automatic TLS certificate provisioning in Knative, you can append the
> `-l networking.knative.dev/certificate-provider!=cert-manager` selector flag to your `kubectl apply` command to exclude 
> the related plugins from your installation.  [Learn more about automatic TLS certificate provisioning](../serving/using-an-ssl-cert.md).

Reply:
We can use below command to delete `networking-certmanager` component from Knative serving:
```
kubectl  delete --filename https://github.com/knative/serving/releases/download/v0.6.0/serving.yaml  -l networking.knative.dev/certificate-provider=cert-manager
```

Reply:
I updated this part by using `kubectl delete`. If it looks good to you, I can apply it to other files.